djautotask/models.py

```diff
@@ -372,6 +375,20 @@ class TaskNote(TimeStampedModel, Note):
     )
 
 
+class Contact(TimeStampedModel):
+
+    first_name = models.TextField(blank=True, null=True, max_length=20)
```

Comment:
Since the first_name and last_name fields only have a max length of 20 these fields should be `CharField`s https://docs.djangoproject.com/en/3.1/ref/models/fields/#charfield

Reply:
Excellent point.